nipype/interfaces/freesurfer/tests/test_auto_Binarize.py

```diff
@@ -1,6 +1,6 @@
 # AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
 from __future__ import unicode_literals
-from ..model import Binarize
+from ..model_bkp import Binarize
```

Comment:
Something seems to have gone wrong. I'm guessing `model_bkp.py` is a backup file you have in your local repository? That's interfering with the proper behavior of `make specs`. Could you remove it, re-run `make specs` and add the updated auto tests?

Reply:
Ah, right. I forgot that I made a backup and was looking at the error message with no clue to what's going on. I did as you suggested, hope that will work.